bib/internal/setup/setup.go

```diff
@@ -148,3 +149,17 @@ func validateCanRunTargetArch(targetArch string) error {
 
 	return nil
 }
+
+func ValidateHasContainerTags(imgref string) error {
+	output, err := exec.Command("podman", "image", "inspect", imgref, "--format", "'{{.Labels}}'").Output()
```

Comment:
(super nitpick) I don't think the `'` around `{{.Labels}}` here is needed as this is not run in the shell but via execve() so no need to quote (but it does also not hurt).